skill/api.py

```diff
@@ -77,15 +77,37 @@
 )
 
 
+def own_language(lang: str):
+    """
+    OWM supports 31 languages, see https://openweathermap.org/current#multi
+
+    Convert Mycroft's language code to OpenWeatherMap's, if missing use english.
+
+    Args:
+        language_config: The Mycroft language code.
+    """
+    special_cases = {"cs": "cz", "ko": "kr", "lv": "la"}
+    lang1, lang2 = lang.split('-')
```

Comment:
There's a constant debate about verbose vs short naming.

I'm wondering if we can sidestep that discussion and be more explicit about what they actually are eg:
- language_primary
- language_subtag
?

Reply:
You mean renaming ```lang1``` and ```lang2``` to ```lang_primary``` and ```lang_subtag```?